azerothcore/azerothcore-wotlk
# fix(db/creature): Various Valiance Keep NPC Cosmetic Updates

## Changes Proposed:
-  Valiance Keep Footmen on the beach in front of Valiance Keep now stay in place in the ready1H position, instead of wondering around like idiots
-  Valiance Keep Defenders now have their weapons unsheathed
-  Valiance Keep Riflemen on the rafters will now shoot at the invisible trigger NPCs and their positions have been corrected so they do not hover in the air
- Remove 1 waypoint attached to 1 GUID of Scourged Flamespitter to prevent it from wondering around when the others have no waypoint/don't do anything else

## Issues Addressed:
1. Valiance Keep Footmen on the beach were wondering around
2. Valiance Keep Defenders weren't prepared for battle
3. Valiance Keep Riflemen allowed the Flamespitters to just spew fire in their faces without any defense, they were also hovering in the air like madmen
4. A Scourged Flamespitter kept thinking he was in charge and led the attack by himself on the keep.  He has been put in check.

## Tests Performed:
1. Applied SQL and watched the changes

## How to Test the Changes:
1. Apply sql, create any character
2. .tele valiancekeep and go to the beach area
3. Observe the Flamespitters and ensure that they don't wonder around
4. Observe the Riflemen on the rafters and ensure they're shooting at the Flamespitters (it's really an invisible NPC that they're shooting at)
5. Observe the Footmen on the beach, if they're not attacking something they should be in one position, in the Ready1H emote
6. Look for the Valiance Keep Defenders, make sure their weapons are out.


## Known Issues and TODO List:
1. I'm pretty sure the Valiance Keep Defender isn't supposed to be here unless Horde players are attacking.  Maybe a CPP thing or SAI to spawn only when hostile players are in the area
2. Some of the Flamespitters freeze after shooting their fire at the keep - not sure how to fix this

## Target Branch(es):
